app/models/person.rb

```diff
@@ -0,0 +1,6 @@
+class Person < ActiveRecord::Base
+  attr_accessor :task_id
+
+  scope :all_tasks, -> { all }
```

Comment:
Is this intentional? It looks like it's unused and might just be a copy/paste error...